### PR TITLE
chore: update aries-framework-go dependency

### PR DESCRIPTION
**Pull request by Moopli**
Signed-off-by: Filip Burlacu <filip.burlacu@securekey.com>

### CODE REVIEW

File: pkg/internal/cryptosetup/cryptosetup_test.go [hunk 1]
```diff
@@ -187,3 +187,16 @@ func (m mockKeyManager) Get(keyID string) (interface{}, error) {
...
+}
+
+func (m mockKeyManager) PubKeyBytesToHandle(pubKey []byte, keyType kmsservice.KeyType) (interface{}, error) {
+	return nil, nil
+}
+
+func (m mockKeyManager) ImportPrivateKey(
+	privKey interface{}, kt kmsservice.KeyType, opts ...kmsservice.PrivateKeyOpts) (string, interface{}, error) {
+	return "", nil, nil
+}
```
Comment (DRK3) [inline pkg/internal/cryptosetup/cryptosetup_test.go:202]: Is this still needed now that you're using the other mockkms?

Comment (Moopli) [reply pkg/internal/cryptosetup/cryptosetup_test.go:202]: I'm using this in a few places, to replicate the same error that your tests were checking for.

**Comment by Moopli** [comment]
Just updating the aries-framework-go dependency again since there have been a lot of commits today, then it's ready to merge.

**Comment by Moopli** [comment]
^ done.

File: pkg/internal/mock/kms/kms.go [hunk 2]
```diff
@@ -9,7 +9,6 @@ package kms
 import (
```
Comment (fqutishat) [inline pkg/internal/mock/kms/kms.go:9]: Delete this file not needed

**Pull request merged**